package.json

```diff
@@ -22,61 +22,39 @@
     "lasttranslator": "Themeisle Translate Team <friends@themeisle.com>"
   },
   "scripts": {
-    "release": "npx semantic-release",
     "start": "npm-run-all --parallel start:*",
-    "start:stock": "webpack --watch --mode development --config my-stock.webpack.config.js",
-    "start:dash": "wp-scripts start dashboard/src/dashboard.js --output-path=dashboard/build --config dash.webpack.config.js",
-    "start:template": "wp-scripts start obfx_modules/template-directory/src/template-directory.js --output-path=obfx_modules/template-directory/js --config dash.webpack.config.js",
-    "i18n": "grunt i18n",
-    "dist": "bash bin/dist.sh",
+    "start:dash": "wp-scripts start dashboard/src/dashboard.js --output-path=dashboard/build --hot --allowed-hosts all --config webpack.config.js",
+    "start:template": "wp-scripts start obfx_modules/template-directory/src/template-directory.js --output-path=obfx_modules/template-directory/js --hot --allowed-hosts all --port 8081 --config webpack.config.js",
+    "start:stock": "wp-scripts start obfx_modules/mystock-import/js/src/registerPlugin.js --output-path=obfx_modules/mystock-import/js/build --hot --allowed-hosts all --port 8082 --config webpack.config.js",
     "build": "npm-run-all build:*",
-    "build:dash": "wp-scripts build dashboard/src/dashboard.js --output-path=dashboard/build --config dash.webpack.config.js",
-    "build:template": "wp-scripts build obfx_modules/template-directory/src/template-directory.js --output-path=obfx_modules/template-directory/js --config dash.webpack.config.js",
-    "build:stock": "webpack --mode production --config my-stock.webpack.config.js"
+    "build:dash": "wp-scripts build dashboard/src/dashboard.js --output-path=dashboard/build --config webpack.config.js",
+    "build:template": "wp-scripts build obfx_modules/template-directory/src/template-directory.js --output-path=obfx_modules/template-directory/js --config webpack.config.js",
+    "build:stock": "wp-scripts build obfx_modules/mystock-import/js/src/registerPlugin.js --output-path=obfx_modules/mystock-import/js/build --config webpack.config.js",
+    "release": "npx semantic-release",
+    "i18n": "grunt i18n",
+    "dist": "bash bin/dist.sh"
   },
   "devDependencies": {
     "@semantic-release/changelog": "^5.0.1",
     "@semantic-release/exec": "^5.0.0",
     "@semantic-release/git": "^9.0.0",
-    "@wordpress/block-editor": "^7.0.0",
-    "@wordpress/components": "^15.0.0",
-    "@wordpress/element": "^4.0.0",
-    "@wordpress/eslint-plugin": "^9.1.0",
-    "@wordpress/hooks": "^3.2.0",
-    "@wordpress/i18n": "^4.2.1",
-    "@wordpress/scripts": "^17.1.0",
-    "babel-core": "^6.25.0",
-    "babel-loader": "^8.2.2",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.6.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
+    "@wordpress/scripts": "^26.19.0",
```

Comment:
You can upgrade this up to `27.9.0` https://github.com/WordPress/gutenberg/blob/b94e18a09b072b67e59005b5424a13081f51d074/packages/scripts/CHANGELOG.md#2790-2024-05-16 since you are using `Node >= 18`

Reply:
I know, except that would mean also including the runtime script for hot-reload. As this would be a small release, I think it's enough of an upgrade for now.